CMakeLists.txt

```diff
@@ -117,6 +115,37 @@ target_include_directories(musica
 
 add_subdirectory(src)
 
+#
+# jiwon 7/21 - in progress
+#
+####################
+# MUSICA - fortran
+add_library(musica-fortran STATIC src/fortran_interface/micm/interface.F90)
+add_library(musica::musica-fortran ALIAS musica-fortran)
+
+set_target_properties(musica-fortran PROPERTIES
+  ARCHIVE_OUTPUT_DIRECTORY ${MUSICA_LIB_DIR}
+  Fortran_MODULE_DIRECTORY ${MUSICA_MOD_DIR}
+)
+
+target_link_libraries(musica-fortran 
+  PUBLIC
+    PkgConfig::netcdff
+    ${JSON_LIB}
+)
+
+target_include_directories(musica-fortran
+  PUBLIC  
+    $<BUILD_INTERFACE:${MUSICA_MOD_DIR}>
+    $<INSTALL_INTERFACE:${INSTALL_MOD_DIR}>
+)
```

Comment:
We may not end up needing this, not sure yet.

Reply:
Comments Added for this